src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1971,9 +2000,12 @@ bool isRefEscape
             Debug.Assert(AllParametersConsideredInEscapeAnalysisHaveArguments(argsOpt, parameters, argsToParamsOpt));
 #endif
 
+            MethodInfo localMethodInfo = methodInfo;
```

Comment:
>methodInfo [](http://example.com/codeflow?start=41&length=10)

Is there a specific reason why we keep the original value accessible in the rest of the method through this parameter? This might be bug prone in the long run.

Reply:
Resolved offline